tasks/update_appraisal_gemfiles.rake

```diff
@@ -15,22 +15,35 @@ TRACER_VERSIONS = %w[
 ].freeze
 # ADD NEW RUBIES HERE
 
+FORCE_BUNDLER_VERSION = {
+  '2.3' => '1.17.3', # Some groups require bundler 1.x https://github.com/DataDog/dd-trace-rb/issues/2444
+  'jruby-9.2.8.0' => '2.3.6', # 2.3.26 seems to be broken https://github.com/DataDog/dd-trace-rb/issues/2443
```

Comment:
Surprising, I was able to run with 2.3.26 locally without issue.

Reply:
Good point. I retried it locally, both with `JRUBY_OPTS=--dev` as well as without (because the default in our docker-compose is without, but on CI is with that setting) and I could not reproduce it.

Since the workaround seems simple, I'd still leave it in (we've seen it fail, so we know it can fail at least rarely), but if you're not convinced I'm happy to revert it.

Reply:
No problem with keeping it, it just feels odd so we may not have been down to the bottom of this.

Reply:
Well, that's fair but... I'm not sure it's worth getting to the bottom of this.

The issue was caused by a `NullPointerException` inside JRuby. That right there should never happen, and suggests an issue with JRuby.

And this issue is on 9.2.8.0, which is known to be very buggy, which is why where are 13 other releases in the 9.2 series, over the span of 3 years.

So my thinking is -- it's quite an investment to chase this down, and in the end we may learn that it's been fixed by a later version anyway, and we still will need to keep 9.2.8.0 around for now :/